kisanola/tech-community-frontend
# add related posts on single post page

#### What does this PR do?
This PR adds related posts when viewing a single post

#### Description of Task to be completed?
• handle when no post is related
• remove the post itself to be repeated on the related ones
• redesign single post page to make place for the new block

#### How should this be manually tested?
• checkout to this branch `ft-related-posts`
• Click on one of the posts
• you will notice a new block on the right side displaying related posts

#### Screenshots (if appropriate)
# 1
![image](https://user-images.githubusercontent.com/27511264/67329979-eec01a80-f51b-11e9-92c7-e695de700c66.png)
# 2
![image](https://user-images.githubusercontent.com/27511264/67330027-00092700-f51c-11e9-8676-75a11735cc98.png)


#### Questions:
